packages/jaeger-ui/src/components/DeepDependencies/Header/LayoutSettings/index.test.js

```diff
@@ -48,6 +48,26 @@ describe('LayoutSettings', () => {
     jest.clearAllMocks();
   });
 
+  it('uses value found in localStorage', () => {
+    const selectedDensity = props.density;
+    localStorage.setItem(LayoutSettings.DEFAULT_DENSITY, selectedDensity);
+    const radios = getWrapper().find(Radio);
+    expect(Array.from(radios).findIndex(radio => radio.props.checked)).toBe(densityIdx);
+  });
```

Comment:
this test is confusing to me. First, you're setting local storage to value that is already selected. A logical test would be:
  1. set local storage to null, initialize component, validate that it sets the option to some default value
  2. set local storage to another value (different from default), initialize the component, validate that it reused the value from local storage